packages/blockchain-info-components/src/Form/PasswordInput.js

```diff
@@ -61,16 +95,32 @@ class PasswordInput extends React.Component {
   }
 
   render () {
-    const { errorState, ...rest } = this.props
-    const borderColor = selectBorderColor(
-      this.props.controlledBorderColor || errorState
+    const {
+      active,
+      controlledBgColor,
+      controlledBorderColor,
```

Comment:
It would be good to clean this up, if we can be consistent with all inputs and don't need a `controlledBorderColor` override lets completely remove this.